documentation-website/Writerside/topics/Deep-Dive-into-DAO.md

```diff
@@ -173,6 +173,32 @@ class User(id: EntityID<Int>): IntEntity(id) {
 }
 ```
 
+### Ordered reference
+
+You can also define the order in which referenced entities appear:
+
+```kotlin
+class User(id: EntityID<Int>) : IntEntity(id) {
+    companion object : IntEntityClass<User>(Users)
+
+    ...
+    val ratings by UserRating referrersOn UserRatings.user orderBy UserRatings.value
```

Comment:
I wonder if we can have an example with `.` chaining as well, it should be more discoverable with completion